src/vuestic-theme/vuestic-components/vuestic-datatable/datatable-components/FilterBar.vue

```diff
@@ -18,36 +18,35 @@ export default {
     label: {
       type: String,
     },
+    value: ''
   },
   data () {
-    return {
-      filterText: '',
-      typingTimeout: null,
-      typingDelay: 250,
-    }
-  },
-  methods: {
-    doFilter () {
-      this.typingTimeout && clearTimeout(this.typingTimeout)
-      this.typingTimeout = setTimeout(() => this.$emit('filter', this.filterText), this.typingDelay)
-    },
+    return {}
   },
+  methods: {},
+  computed: {
+    valueProxy: {
```

Comment:
@asvae , why using value proxy here?

Reply:
@jean-moldovan any other sensible way to work with input?